### ISSUE TITLE
Podcast/gPodder support?

### ISSUE THREAD

**Issue by tgp1994** [theweavrs/BreadPlayer#239]
I'd love to see an open source, C# podcast player (and even cooler if it can integrate with gPodder!) I'd like to help too. I develop in C# and maybe more importantly, I really want to see a modern player with this. :) 

Would this fit with the direction of your project? I've been thinking of doing a little programming, so I can fork it soon and start working on this, and do a pull request later. Thank you!

**Comment by theweavrs** [comment]
Yes, I did plan on adding podcast support. You are welcome to help. :)
